webproject/src/main/java/com/google/sps/util/FirebaseUtil.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.util;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+
+public class FirebaseUtil {
```

Comment:
this FirebaseUtil should be:
1. Injectable
2. has function to do GET/POST/DELETE data from firebase
3. run initializeFirebase() in the constructor, which will be called only once, being an injectable as it is

Reply:
Okay! I have added logic to fetch whether a user has already joined a room & to add a new UserRoom